Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,35 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    context "when f.plist is not nil" do
+      it "prints plist startup information when f.plist_startup is not nil" do
+        allow(f).to receive(:plist).and_return(true)
+        allow(f).to receive(:plist_startup).and_return(true)
```

Comment:
This is how plist is defined in formula.rb. So, that's why I'm mocking `f.plist`.
```
def plist
    nil
end```
For `plist_startup`, I'm trying to set it as true through `plist_options`: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L2361
Do you think a `formula do` will still be needed?